include/GL/glui.h

```diff
@@ -496,35 +549,34 @@ class GLUIAPI GLUI_Master_Object
     void (*glut_passive_motion_CB)(int,int);
     void (*glut_visibility_CB)(int);
     void (*glut_motion_CB)(int,int);
-    void (*glut_display_CB)(void);
+    void (*glut_display_CB)();
     void (*glut_entry_CB)(int);
     **********/
 
     void  set_left_button_glut_menu_control( GLUI_Control *control );
 
     /********** GLUT callthroughs **********/
-    /* These are the glut callbacks that we do not handle */
-
+    
     void set_glutReshapeFunc (void (*f)(int width, int height));
     void set_glutKeyboardFunc(void (*f)(unsigned char key, int x, int y));
     void set_glutSpecialFunc (void (*f)(int key, int x, int y));
     void set_glutMouseFunc   (void (*f)(int, int, int, int ));
 
-    void set_glutDisplayFunc(void (*f)(void)) {glutDisplayFunc(f);}
-    void set_glutTimerFunc(unsigned int millis, void (*f)(int value), int value)
-    { ::glutTimerFunc(millis,f,value);}
-    void set_glutOverlayDisplayFunc(void(*f)(void)){glutOverlayDisplayFunc(f);}
-    void set_glutSpaceballMotionFunc(Int3_CB f)  {glutSpaceballMotionFunc(f);}
-    void set_glutSpaceballRotateFunc(Int3_CB f)  {glutSpaceballRotateFunc(f);}
-    void set_glutSpaceballButtonFunc(Int2_CB f)  {glutSpaceballButtonFunc(f);}
-    void set_glutTabletMotionFunc(Int2_CB f)        {glutTabletMotionFunc(f);}
-    void set_glutTabletButtonFunc(Int4_CB f)        {glutTabletButtonFunc(f);}
-    /*    void set_glutWindowStatusFunc(Int1_CB f)        {glutWindowStatusFunc(f);} */
-    void set_glutMenuStatusFunc(Int3_CB f)            {glutMenuStatusFunc(f);}
-    void set_glutMenuStateFunc(Int1_CB f)              {glutMenuStateFunc(f);}
-    void set_glutButtonBoxFunc(Int2_CB f)              {glutButtonBoxFunc(f);}
-    void set_glutDialsFunc(Int2_CB f)                      {glutDialsFunc(f);}  
-  
+	/* These are the glut callbacks that we do not handle */
+
+    void set_glutDisplayFunc(void (*f)());
+    void set_glutTimerFunc(unsigned int millis, void (*f)(int value), int value);
+    void set_glutOverlayDisplayFunc(void(*f)());
+    void set_glutSpaceballMotionFunc(Int3_CB f);
+    void set_glutSpaceballRotateFunc(Int3_CB f);
+    void set_glutSpaceballButtonFunc(Int2_CB f);
+    void set_glutTabletMotionFunc(Int2_CB f);
+    void set_glutTabletButtonFunc(Int4_CB f);
+    void set_glutWindowStatusFunc(Int1_CB f);
+    void set_glutMenuStatusFunc(Int3_CB f);
+    void set_glutMenuStateFunc(Int1_CB f);
+    void set_glutButtonBoxFunc(Int2_CB f);
+    void set_glutDialsFunc(Int2_CB f);  
 
```

Comment:
`+` `set_glutMotionFunc(Int2_CB f); `
`+` `set_glutPassiveMotionFunc(Int2_CB f); `

I think these and any others are needed for completeness so wrappers can be applied uniformly.

Perhaps in the future GLUI could use a #define macro to replace GLUT's entrypoints with its own. That would get this out of the class for better abstraction. But for now there should just be all or none.